src/pages/Signup/Signup.scss

```
@@ -1 +1,43 @@
-//q
+@import '../../styles/mixin';
+@import '../../styles/variables';
+
+.main-container {
+  @include flexCenter;
+  flex-direction: column;
+
+  .gron-logo {
+    width: 300px;
+    height: 100px;
+    margin: 70px 50px 70px 0px;
+  }
+  .name-input,
+  .email-input,
+  .password-input,
+  .phone-number-input,
+  .address-input {
+    width: 470px;
+    height: 70px;
+    margin-bottom: 20px;
+    padding: 20px;
+    border: 3px solid $primary-color;
+    border-radius: 10px;
+  }
+  .privacy-agreement-input {
+    font-size: 22px;
+    @include flexCenter;
+    .privacy-agreement-checkbox {
+      zoom: 1.6;
+      accent-color: $primary-color;
+    }
```

```suggestion
    @include flexCenter;
    font-size: 22px;
    
    .privacy-agreement-checkbox {
      zoom: 1.6;
      accent-color: $primary-color;
    }
```
@include는 스타일 블록 가장 최상위에 위치해야 하는 부분입니다!
그리고 선택자끼리는 가독성을 위해 한칸씩 띄워주세요!